resources/brew/brew.dsc.resource.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+export exist=true
+export NONINTERACTIVE=1
+
```

Comment:
you might add a comment about `$packageName` which is expected to be exported in the environment

Reply:
good point, adding